### ISSUE TITLE
Exception thrown in ZstdNet.Compressor destructor when library not found

### ISSUE THREAD

**Issue by ameisen** [ameisen/SV-SpriteMaster#6]
```
[13:19:50 ERROR SMAPI] Critical app domain exception: System.DllNotFoundException: libzstd
  at (wrapper managed-to-native) ZstdNet.ExternMethods:ZSTD_freeCCtx (intptr)
  at ZstdNet.Compressor.Dispose (Boolean disposing) <0x40631af0 + 0x0001f> in <filename unknown>:0
  at ZstdNet.Compressor.Finalize () <0x40631610 + 0x00017> in <filename unknown>:0 
```

This is due to their trying to _unload_ the `libzstd` library... in a situation where it could not load in the first place.

Should be fixed with a Harmony finalizer patch.

**Comment by ameisen** [comment]
Fixed as of 5405fdcac4a3e9c54d36b0514c652f98b30592a6.

**Comment by ameisen** [comment]
Fixed in 5405fdcac4a3e9c54d36b0514c652f98b30592a6.
